src/main/java/appeng/client/gui/implementations/GuiWirelessTerm.java

```diff
@@ -30,7 +30,7 @@
 	public GuiWirelessTerm( final InventoryPlayer inventoryPlayer, final IPortableCell te )
 	{
 		super( inventoryPlayer, te );
-		this.maxRows = Integer.MAX_VALUE;
+		//this.maxRows = Integer.MAX_VALUE;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)